.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
add testing too!!

Reply:
sorry, i did testing husky!!

Reply:
do you mean this : npm run lint && npm run test  ?